src/animations/constructor.js

```diff
@@ -0,0 +1,64 @@
+// Require react with addons to get access to React Transition Groups
+var React = require("react/addons");
+
+//Require arrival to know when elements and their children have transitionended.
```

Comment:
Last word is a typo or needs to be separated out and then sentence restructured a bit.